packages/components/src/font-size-picker/index.native.js

```diff
@@ -23,6 +23,7 @@ const DEFAULT_FONT_SIZE = 16;
 
 function FontSizePicker( {
 	fontSizes = [],
+	// Can this be kept as-is?
```

Comment:
@WordPress/native-mobile @dcalhoun this is still a WIP, but wanted to check with you about the best thing to do here in case we move forward — we're basically considering deprecating the `disableCustomFontSizes` prop in favor of a `pickerMode` prop in the web version.

Reply:
Thank you @dcalhoun 🙏  Should we go ahead with the PR, I will apply your suggested changes and ping you again for review